docs/python/editing.md

```diff
@@ -22,45 +22,46 @@ IntelliSense is a general term for code editing features that relate to code com
 
-[Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) offers auto import suggestions for modules in your workspace and/or packages you have installed in your environment. This enables import statements to be conveniently added automatically as you're typing.
 
-By default, only top-level symbols/packages are suggested, but you can customize this behavior through the `python.analysis.packageIndexDepths` setting. See the section below on "Customize IntelliSense behavior" to learn more about IntelliSense controls.
 
 ### Customize IntelliSense behavior
 
 Because enabling the full set of IntelliSense features by default could end up making your development experience feel slower, we enable a minimum set of features that allow you to be productive while still having a performant experience. However, you can customize the behavior of the analysis engine to your liking through multiple settings.
 
-For the full list of available IntelliSense controls, you can reference the Python extension [code analysis settings](/docs/python/settings-reference.md#code-analysis-settings) and [autocomplete settings](/docs/python/settings-reference.md#autocomplete-settings).
-
-You can also customize the general behavior of autocomplete and IntelliSense, even to disable these VS Code features entirely. See [Customizing IntelliSense](/docs/editor/intellisense.md#customizing-intellisense).
-
-![IntelliSense and autocomplete for Python code](images/editing/python-editing.gif)
+#### Enable Auto Imports
+[Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) offers auto import suggestions for modules in your workspace and/or packages you have installed in your environment. This enables import statements to be conveniently added automatically as you're typing. Auto imports are disabled by default, but you can enable it by setting `python.anaysis.autoImportCompletions` to `true` in your settings.
 
-> **Tip**: Check out the [IntelliCode extension for VS Code (preview)](https://go.microsoft.com/fwlink/?linkid=2006060). IntelliCode provides a set of AI-assisted capabilities for IntelliSense in Python, such as inferring the most relevant auto-completions based on the current code context. For more information, see the [IntelliCode for VS Code FAQ](https://learn.microsoft.com/visualstudio/intellicode/intellicode-visual-studio-code).
+By default, only top-level symbols/packages are suggested, but you can customize this behavior through the `python.analysis.packageIndexDepths` setting. User defined symbols (i.e. not coming from installed packages or libraries) are only automatically imported if they have already been used in files that were previously opened in the editor. Otherwise, they will only be available through the [add imports Quick Fix](/docs/python/editing.md#quick-fixes).
```

Comment:
Would it be worth it to add an example of package index depths setting? I guess reading this I'm not entirely sure how it works, so I was thinking an example would help me at least. 

Like what's a top-level symbol/package? First level in a directory? What if I didn't realize python libs are laid down as directory trees?

Reply:
I added a link to the settings reference so they can see an example of how to use it, and added an example of what a top-level package import suggestion would look like, but I'm not sure this is a good one. @heejaechang do you have a better example in mind by any chance?